Dockerfile

```diff
@@ -3,16 +3,16 @@ FROM debian:bullseye
 USER root
 RUN apt-get update -m && \
     apt-get install -y maven openjdk-17-jdk nodejs npm
-RUN npm install -g typescript
+RUN npm install -g typescript@3.9.9
 RUN mkdir /build
 COPY . /build/
-RUN cd /build && \
+RUN --mount=type=cache,target=/root/.m2 cd /build && \
     tsc -p shesmu-server-ui && \
-    mvn -q -Dsurefire.useFile=false -DskipIT=true clean install && \
+    mvn -Dsurefire.useFile=false -DskipIT=true clean install && \
     mkdir /usr/share/shesmu && \
     cp shesmu-pluginapi/target/shesmu-pluginapi.jar shesmu-server/target/shesmu.jar plugin-*/target/shesmu-plugin-*.jar /usr/share/shesmu
 
-FROM openjdk:16-slim
+FROM openjdk:17
```

Comment:
I have yet to verify that shesmu runs correctly with java 17?

Reply:
We don't use the Dockerfile in our local deploy - I believe it's only used to build a Docker image during the release (and that hasn't been working reliably recently).

Reply:
Why update it in this PR?

Reply:
I was testing with the dockerfile locally, I can split this out into another PR for later.